docs/source/nlp/nemo_megatron/batching.rst

```diff
@@ -5,17 +5,37 @@ Batching
 
 Batch size is one of the first parameters you should play with. For efficiency and convergence reasons we recommend you first try maximizing your batch size per GPU so that your GPU RAM usage is maximized.
 
-NeMo Megatron uses the following concepts.
+NeMo Framework uses the following concepts:
 
-*Micro batch size* is the number of examples per data parallel rank. It is controlled by ``model.micro_batch_size`` parameter.
+===========================  ==================================================================================================================================
+Parameter                    Description
+===========================  ==================================================================================================================================
+Micro Batch Size             The number of examples per data parallel rank.
+Global Batch Size            ``global batch size = micro_batch_size * data_parallel_size * gradient_accumulation_steps``. For details on ``data_parallel_size`` see `this page <https://github.com/NVIDIA/NeMo/blob/main/docs/source/features/parallelisms.rst>`_.
+Gradient Accumulation        Supports training with large batch sizes with fixed memory footprint at the cost of additional compute. ``accumulate_grad_batches`` is automatically handled by PyTorch Lightning.
```

Comment:
suggested revision

This parameter supports training with large batch sizes while maintaining a fixed memory footprint, though it requires additional compute. The ``accumulate_grad_batches`` parameter is automatically managed by PyTorch Lightning.